<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setDataSource</class>
 <widget class="QDialog" name="setDataSource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setDataSource</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Publish</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chk_dds_pub">
        <property name="text">
         <string>DDS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_hla_pub">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>HLA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_sock_pub">
        <property name="text">
         <string>Sock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Subscribe</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chk_dds_sub">
        <property name="text">
         <string>DDS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_hla_sub">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>HLA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_sock_sub">
        <property name="text">
         <string>Sock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>setDataSource</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>setDataSource</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
